key_action_map.cpp

```diff
@@ -185,6 +185,31 @@ void save_to_file(const key_action_map& kam, const std::string& filename)
 sf::Keyboard::Key to_sfml_key(const std::string& s)
 {
   if (s == "A") return sf::Keyboard::Key::A;
+  if (s == "B") return sf::Keyboard::Key::B;
+  if (s == "C") return sf::Keyboard::Key::C;
+  if (s == "D") return sf::Keyboard::Key::D;
+  if (s == "E") return sf::Keyboard::Key::E;
+  if (s == "F") return sf::Keyboard::Key::F;
+  if (s == "G") return sf::Keyboard::Key::G;
+  if (s == "H") return sf::Keyboard::Key::H;
+  if (s == "I") return sf::Keyboard::Key::I;
+  if (s == "J") return sf::Keyboard::Key::J;
+  if (s == "K") return sf::Keyboard::Key::K;
+  if (s == "L") return sf::Keyboard::Key::L;
+  if (s == "M") return sf::Keyboard::Key::M;
+  if (s == "N") return sf::Keyboard::Key::N;
+  if (s == "O") return sf::Keyboard::Key::O;
+  if (s == "P") return sf::Keyboard::Key::P;
+  if (s == "Q") return sf::Keyboard::Key::Q;
+  if (s == "R") return sf::Keyboard::Key::R;
+  if (s == "S") return sf::Keyboard::Key::S;
+  if (s == "T") return sf::Keyboard::Key::T;
+  if (s == "U") return sf::Keyboard::Key::U;
+  if (s == "V") return sf::Keyboard::Key::V;
+  if (s == "W") return sf::Keyboard::Key::W;
+  if (s == "X") return sf::Keyboard::Key::X;
+  if (s == "Y") return sf::Keyboard::Key::Y;
+  if (s == "Z") return sf::Keyboard::Key::Z;
```

Comment:
There is probably a very smart alternative to these 26 lines of code, but I enjoy the safety and readability of this option 👍

Reply:
It is clumsy indeed. If someone wants to practice making code smarter, I think this is a fun section to work on :-)